src/Controls/src/SourceGen/CodeBehindGenerator.cs

```diff
@@ -358,7 +358,15 @@ static void GenerateXamlCodeBehind(XamlProjectItem? xamlItem, Compilation compil
 
 		sb.AppendLine("\t\tprivate void InitializeComponent()");
 		sb.AppendLine("\t\t{");
-		sb.AppendLine($"\t\t\tglobal::Microsoft.Maui.Controls.Xaml.Extensions.LoadFromXaml(this, typeof({rootType}));");
+		sb.AppendLine("\t\t\t[global::System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage(\"TrimAnalysis\", \"IL2026:RequiresUnreferencedCode\",");
+		sb.AppendLine("\t\t\t\tJustification = \"The body of InitializeComponent will be replaced by XamlC so LoadFromXaml will never be called in production builds.\")]");
```

Comment:
Maybe the message could be:
```suggestion
		sb.AppendLine("\t\t\t\tJustification = \"By default, Xaml compilation replaces the body of InitializeComponent, so LoadFromXaml will never be called in production builds.\")]");
```